caterva2/services/templates/includes/prompt.html

```diff
@@ -30,11 +30,14 @@ <h5>List of commands</h5>
           {% endfor %}
       </dl>
       <h5>Expressions</h5>
-      <pre>x = &lt;expression&gt;</pre>
+      <pre>x = &lt;expression&gt; # optionaldict</pre>
```

Comment:
Following the eval convention, I'd prefer using `locals` as the name for the current optionaldict.